### PR TITLE
Bump unit test coverage a few percent

### PR DESCRIPTION
**Pull request by redshiftzero**
Adds a new file of unit tests for `db.py` and adds no cover lines where appropriate

### CODE REVIEW

**Comment by heartsucker** [comment]
I think it might be worthwhile to write some super simple tests for the overridden `__repr__` functions instead of `no cover`-ing them because I've actually seen production bugs where some edge case calls a broken `repr` and everything blows up because it was never called for the like 8 months it was in the code base.

**Comment by redshiftzero** [comment]
Ah, great point @heartsucker! I'll revise this.

**Comment by redshiftzero** [comment]
OK simple tests added for `__repr__` lines (and `no cover` lines removed) for the reason @heartsucker mentions above.

(Also note that the coveralls check below is showing the difference since the last build on this branch, not from develop 😿 ... this PR increases coverage 4%)

File: securedrop/tests/test_unit_db.py [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+from sqlalchemy.orm.exc import MultipleResultsFound, NoResultFound
+
+import journalist
+import crypto_util
+import common
+os.environ['SECUREDROP_ENV'] = 'test'
+from db import (db_session, Journalist, Submission, Source, Reply,
+                get_one_or_else)
+
+import logging
```
Comment (fowlslegs) [inline securedrop/tests/test_unit_db.py]: You should move this up to the top for PEP8.

File: securedrop/tests/test_unit_db.py [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+import unittest
+
+from flask_testing import TestCase
+import mock
+from sqlalchemy.orm.exc import MultipleResultsFound, NoResultFound
+
+import journalist
+import crypto_util
+import common
+os.environ['SECUREDROP_ENV'] = 'test'
```
Comment (fowlslegs) [inline securedrop/tests/test_unit_db.py]: You don't need this if you're not importing `config`.

**Comment by fowlslegs** [comment]
Needs rebase.

**Comment by redshiftzero** [comment]
OK, rebased on current develop to use the utility functions, all changes made and mocked the logger. Should be good to go!
